### PR TITLE
Admin docs: Add a section on AI features

### PR DESCRIPTION
**Pull request by marcelklehr**
### ☑️ Resolves

* Nothing :(

### 🖼️ Screenshots

image

image

### CODE REVIEW

**Comment by marcelklehr** [comment]
Anything else I should add?

**Review by bigcat88 — APPROVED** [review]
looks really cool!

**Comment by DaphneMuller** [comment]
maybe use instead of "implementation" > "app name" and provide a URL to the app store

**Comment by DaphneMuller** [comment]
i love the table, the explanation you made of the ethical AI rating, and the extra explanation of that there are multiple apps implementing the same feature. I can see you put a lot of work in it, thanks a lot

File: admin_manual/ai/index.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Features
+--------
+
+Some of our AI features are realized as generic APIs that any app can provide an implementation for by registering a provider. These are
+Machine translation, Speech-To-Text and Text processing
+
+Machine translation
+^^^^^^^^^^^^^^^^^^^
+As you can see in the table above we have multiple apps offering machine translation capabilities. Each app brings its own set of supported languages.
+In downstream apps like the Text app, users can use the translation functionality regardless of which app implements it behind the scenes.
```
Comment (julien-nc) [inline admin_manual/ai/index.rst:45]:
```suggestion
In downstream apps like the Text app, users can use the translation feature regardless of which app implements it behind the scenes.
```
Nitpicking, optional change :grin:

**Review by DaphneMuller — APPROVED** [review]

File: admin_manual/ai/index.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Until Hub 3, we succeeded in offering features without reliance on proprietary blobs or third party servers. Yet, while there is a large community developing ethical, safe and privacy-respecting technologies, there are many other relevant technologies users might want to use. We want to provide users with these cutting-edge technologies – but also be transparent. For some use cases, ChatGPT might be a reasonable solution, while for other data, it is paramount to have a local, on-prem, open solution. To differentiate these, we developed an Ethical AI Rating.
+
+The rating has four levels:
+ * Red 🔴
+ * Orange 🟠
+ * Yellow 🟡
+ * Green 🟢
+
+It is based on points from these factors:
+ * Is the software (both for inferencing and training) open source?
```
Comment (julien-nc) [inline admin_manual/ai/index.rst:63]:
```suggestion
 * Is the software (both for inferencing and training) under a free and open source license?
```

**Review by julien-nc — CHANGES_REQUESTED** [review]
:blue_heart: :+1:

File: admin_manual/ai/index.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+   "Smart inbox","mail","Green 🟢","✅","✅","✅","✅"
+   "Image object recognition","recognize","Green 🟢","✅","✅","✅","✅"
+   "Image face recognition","recognize","Green 🟢","✅","✅","✅","✅"
+   "Video action recognition","recognize","Green 🟢","✅","✅","✅","✅"
+   "Audio music genre recognition","recognize","Green 🟢","✅","✅","✅","✅"
+   "Suspicious login detection","suspicious_login","Green 🟢","✅","✅","✅","✅"
+   "Related resources","related_resources","Green 🟢","✅","✅","✅","✅"
+   "Recommended files","recommended_files","Green 🟢","✅","✅","✅","✅"
+   "Machine translation","translate","Green 🟢","✅","✅ - Opus models by University Helsinki","✅","✅"
```
[the remark this answers is not in the document]
Comment (marcelklehr) [reply admin_manual/ai/index.rst]: Yes :)

File: admin_manual/ai/index.rst (hunk 1 above)
Comment (marcelklehr) [reply admin_manual/ai/index.rst:45]: :D

**Comment by marcelklehr** [comment]
Do we backport this to stable27?

**Comment by DaphneMuller** [comment]
> Do we backport this to stable27?

up to you to decide  💙

**Pull request merged**